systems/plants/@RigidBodyManipulator/pointAcceleration.m

```diff
@@ -0,0 +1,42 @@
+function [point_accel, dpoint_accel] = pointAcceleration(obj, kinsol, body_index, point)
```

Comment:
is this really a method that we want to have a public face?  i worry that it is potentially confusing.  i preferred it back inside centerOfMassJacobianDotTimesV.  if it is going to be used by multiple methods, then we could at least make it private?